app/assets/stylesheets/govuk_publishing_components/components/_subscription-links.scss

```diff
@@ -0,0 +1,40 @@
+.gem-c-subscription-links {
+  @include bold-19;
+
+  .gem-c-subscription-links__list {
+    list-style: none;
+    margin-left: -$gutter-half / 2;
+    margin-right: -$gutter-half / 2;
+  }
+
+  .gem-c-subscription-links__list-item {
+    display: inline-block;
+    margin-left: $gutter-half / 2;
+    margin-right: $gutter-half / 2;
+    margin-bottom: $gutter / 2;
+  }
+
+  .gem-c-subscription-links__link {
+    text-decoration: none;
+    padding-left: 28px;
+    background-repeat: no-repeat;
+    background-position: 0 20%;
+
+    @include media(tablet) {
+      background-position: 0 35%;
+    }
+  }
+
+  .gem-c-subscription-links__link--feed {
+    background-image: image-url("feed-icon-black.png");
```

Comment:
We need to copy over these icons into the gem as well

Reply:
Ah, can't believe I missed that. Actually on second thought I can, worth noting... I think I missed this because I was moving the gem while using the component guide in government-frontend for reference, so it was still picking up the images from government-frontend. Fixed.